salt/states/nfs_export.py

```diff
@@ -0,0 +1,214 @@
+# -*- coding: utf-8 -*-
+'''
+Management of NFS exports
+===============================================
+
```

Comment:
Can you add a `.. versionadded:: Oxygen` tag here?

Reply:
Should I add any other metadata, like Platform?

Reply:
You can if you'd like, but it isn't required. :)